backend/.env.dev

```diff
@@ -2,16 +2,18 @@
 DOMAIN=localhost
 ENVIRONMENT=local
 PROJECT_NAME="Districtr v2 backend"
-BACKEND_CORS_ORIGINS="http://localhost,http://localhost:5173"
-SECRET_KEY={fill-me}
+BACKEND_CORS_ORIGINS="http://localhost,http://localhost:5173,http://localhost:3000,http://127.0.0.1:3000"
+SECRET_KEY="ASDFASDFASDF"
+LOAD_GERRY_DB_DATA=true
+GPKG_DATA_DIR=sample_data
 
 # Postgres
-DATABASE_URL=postgresql+psycopg://postgres:{fill-me}@localhost:5432/districtr
+DATABASE_URL=postgresql+psycopg://postgres:postgres@localhost:5432/districtr
 POSTGRES_SCHEME=postgresql+psycopg
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD={fill-me}
+POSTGRES_PASSWORD=postgres
 POSTGRES_DB=districtr
-POSTGRES_SERVER=localhost
+POSTGRES_SERVER=db
```

Comment:
H: These will vary by user. Can we set these env vars in the docker compose rather than changing them here?

Reply:
Done!